docs/quickstart/apollo_software_installation_guide.md

```diff
@@ -80,6 +80,11 @@ bash docker/scripts/dev_into.sh
 From the `/apollo` directory inside Apollo Docker container, type:
 
 ```bash
+# for AMD gpu this should be run every time after you run
```

Comment:
Is this caused by the container or will it be fixed?

Reply:
It can be fixed; something to do how the Docker image is generated. When we do `dev_start` for the first time, reinstall the RPP, commit the image and `dev_start` with this image commit, RPP works fine.

Reply:
image commit is not recommended, we better be able to fix this!

Reply:
Agreed. We are working with RPP team to fix this. 

The workaround for now is to rebuild RPP after dev_start.  Could we please have the first version with this workaround?

Reply:
yes, we should add a todo commit in the code, so we don't forget about the issue

Reply:
Agreed. We will add it.

Reply:
@GowthamKudupudi  Please add a todo commit to the code.

Reply:
Done
https://github.com/amd/apollo/commit/b05b1a5f01d088a17b0d64bc762b987b90b40e8b